tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -277,8 +288,43 @@ <h3>No scalar data was found.</h3>
         return _smoothingWeight > 0;
       },
 
-      _shouldOpen(index) {
-        return index <= 2;
+      _getCategoryKey(category) {
+        return category.metadata.type ==
+            tf_categorization_utils.CategoryType.SEARCH_RESULTS ?
+                'search' : category.name;
```

Comment:
'search' could match an existing category name, couldn't it?  Probably better to pick something less likely to collide like '$search' or even just the empty string.